new_lineage_panel/src/CustomNodes.tsx

```diff
@@ -461,3 +583,47 @@ export const ColumnNode: FunctionComponent<NodeProps> = ({ data }) => {
     </div>
   );
 };
+
+const LIGHT_ICONS: Record<string, React.ReactNode> = {
```

Comment:
each of these icons differ by fill color. We can use `currentColor` property in fill and handle the colors using css with theme. This will avoid maintaining multiple svgs and these Light and dark icon objects